pennylane/measurements/classical_shadow.py

```diff
@@ -228,14 +230,28 @@ class ClassicalShadowMP(MeasurementTransform):
 
 
     Args:
-        args (tuple[Any]): Positional arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
+        wires (.Wires): The wires the measurement process applies to.
         seed (Union[int, None]): The seed used to generate the random measurements
-        kwargs (dict[Any, Any]): Additional keyword arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
+        id (str): custom label given to a measurement instance, can be useful for some applications
+            where the instance has to be identified
     """
 
-    def __init__(self, *args, seed=None, **kwargs):
+    def __init__(
+        self, wires: Optional[Wires] = None, seed: Optional[int] = None, id: Optional[str] = None
+    ):
         self.seed = seed
-        super().__init__(*args, **kwargs)
+        super().__init__(wires=wires, id=id)
+
+    @property
+    def hash(self):
+        """int: returns an integer hash uniquely representing the measurement process"""
+        fingerprint = (
```

Comment:
I am not an expert with classical shadows. @Qottmann  Is this fingerprint sufficient to represent the CS measurement process ?

Reply:
I'm taking this list from the call signature of `classical_shadow`.

Reply:
Don't see a problem 👍